README.md

```diff
@@ -116,6 +116,8 @@ carthage update Box Result
 
 If the framework you want to add to your project has dependencies explicitly listed in a [Cartfile][], Carthage will automatically retrieve them for you. You will then have to **drag them yourself into your project** from the [Carthage/Build] folder.
 
+If the embedded framework in your project has dependencies to other frameworks you must list them in the [Cartfile][], and **link them to application target** (even if application target does not have dependency to that frameworks and never uses them).
```

Comment:
But I do not have a (separate) `Cartfile` for my embedded framework project. Should I create one, add my `Other.framework` to it and then, have my app project's `Cartfile` without `Other.framework` record?